### ISSUE TITLE
Add bitlum.io to wallets page

### ISSUE THREAD

**Issue by thorie7912** [thorie7912/microlancer#114]
Also add link to wallets page on QR invoice page for new users

**Comment by AndrewSamokhvalov** [comment]
Hey! Are you still planning to add wallet?

**Comment by thorie7912** [comment]
Yes, I am currently re-working the UI though. Once that is done, I'll get to the next items. Sorry for all the delays, it's tough as I am the only developer :)

**Comment by AndrewSamokhvalov** [comment]
Cool, thanks : )
